woocommerce/woocommerce-admin
# Update default dashboard view.

Fixes #3998.

This PR seeks to further reduce the default dashboard display.

### Detailed test instructions:

- Delete your `woocommerce_admin_dashboard_sections` user meta, if it's set.
- Clear object cache if present
- Go to WooCommerce > Dashboard
- Verify that the default widgets match this screenshot:

![](https://user-images.githubusercontent.com/4500952/77579592-cd32d880-6e97-11ea-99b3-ef881cf3b45e.png)



### Changelog Note:



Tweak: Reduce the number of default widgets on the dashboard.
